ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -47,17 +48,17 @@ class FaultInjector
     VelocityLimit=7, 
     NoForceData=8};
 
-  enum SystemFaults {
-    System=1, 
-    ArmGoalError=2, 
-    ArmExecutionError=4,
-    TaskGoalError=8, 
-    CamGoalError=16, 
-    CamExecutionError=32, 
-    PtGoalError=64, 
-    PtExecutionError=128, 
-    LanderExecutionError = 256};
-
+  std::bitset<9> isNominal{};
+	std::bitset<9> isSystem{		0b0'0000'0001 };
+	std::bitset<9> isArmGoalError{		0b0'0000'0010 };
+	std::bitset<9> isArmExecutionError{		0b0'0000'0100 };
+	std::bitset<9> isTaskGoalError{	0b0'000'1000 };
+	std::bitset<9> isCamGoalError{	0b0'0001'0000 };
+	std::bitset<9> isCamExecutionError{	0b0'0010'0000 };
+	std::bitset<9> isPtGoalError{		0b0'0100'0000 };
+	std::bitset<9> isPtExecutionError{	0b0'1000'0000 };
+	std::bitset<9> isLanderExecutionError{	0b1'0000'0000 };
```

Comment:
Use *constexpr* to make sure no one inadvertently modifies these constants in other parts of the code. 
```suggestion
	static constexpr std::bitset<9> isSystem{		0b0'0000'0001 };
	static constexpr std::bitset<9> isArmGoalError{		0b0'0000'0010 };
	static constexpr std::bitset<9> isArmExecutionError{		0b0'0000'0100 };
	static constexpr std::bitset<9> isTaskGoalError{	0b0'000'1000 };
	static constexpr std::bitset<9> isCamGoalError{	0b0'0001'0000 };
	static constexpr std::bitset<9> isCamExecutionError{	0b0'0010'0000 };
	static constexpr std::bitset<9> isPtGoalError{		0b0'0100'0000 };
	static constexpr std::bitset<9> isPtExecutionError{	0b0'1000'0000 };
	static constexpr std::bitset<9> isLanderExecutionError{	0b1'0000'0000 };
```